library/psa_crypto_driver_wrappers.c

```diff
@@ -1066,4 +1071,172 @@ psa_status_t psa_driver_wrapper_cipher_abort(
 #endif /* PSA_CRYPTO_DRIVER_PRESENT */
 }
 
+/*
+ * Hashing functions
+ */
+psa_status_t psa_driver_wrapper_hash_compute(
+    psa_algorithm_t alg,
+    const uint8_t *input,
+    size_t input_length,
+    uint8_t *hash,
+    size_t hash_size,
+    size_t *hash_length)
+{
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+
+    /* Try accelerators first */
+#if defined(PSA_CRYPTO_DRIVER_TEST) && defined(MBEDTLS_PSA_ACCEL_HASH)
```

Comment:
Eventually, just remove MBEDTLS_PSA_ACCEL_HASH I think, see comments in psa_crypto_hash.c.

Reply:
Which is either going to cause an undefined reference for the test driver entry point (if the entry points keep being guarded with DRIVER_TEST && ACCEL_HASH), or internally inside psa_crypto_hash.c. Unless you are trying to say that the hash core needs to be present on PSA_CRYPTO_DRIVER_TEST? Isn't that overreaching a bit?